front-end/js/script.js

```diff
@@ -1,27 +1,43 @@
 function initMap() {
     var map = new google.maps.Map(document.getElementById('map'), {
-        zoom: 7,
-        center: locations[0]
+        zoom: 7
     });
-    // Create an array of alphabetical characters used to label the markers.
+
     var labels = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
-    // Add some markers to the map.
-    // Note: The code uses the JavaScript Array.prototype.map() method to
-    // create an array of markers based on a given "locations" array.
-    // The map() method here has nothing to do with the Google Maps API.
-    var markers = locations.map(function (location, i) {
-        return new google.maps.Marker({
-            position: location,
+
+    var latlngbounds = new google.maps.LatLngBounds();
+
+    for (i = 0; i < locations.length; i++) {
```

Comment:
how about 'location' instead of 'i'? To be more meaningful.

Reply:
Modified it